oas_docs/output/kibana.serverless.staging.yaml

```diff
@@ -8096,6 +8096,7 @@ paths:
         - Security Endpoint Management API
   /api/endpoint/suggestions/{suggestion_type}:
     post:
+      deprecated: true
```

Comment:
curious... I'm not familiar with these files... How/where are they used?

Reply:
https://www.elastic.co/docs/api/doc/kibana

Reply:
Thanks. I'll start a conversation in our channel